cmd/src/search.go

```diff
@@ -549,12 +598,13 @@ const searchResultsTemplate = `{{- /* ignore this line for template formatting s
 
 			{{- /* Line matches */ -}}
 			{{- $lineMatches := .lineMatches -}}
+			{{- $content := .file.content -}}
 			{{- range $index, $match := $lineMatches -}}
 				{{- if not (searchSequentialLineNumber $lineMatches $index) -}}
 					{{- color "search-border"}}{{"  ------------------------------------------------------------------------------\n"}}{{color "nc"}}
 				{{- end -}}
 				{{- "  "}}{{color "search-line-numbers"}}{{pad (addFloat $match.lineNumber 1) 6 " "}}{{color "nc" -}}
-				{{- color "search-border"}}{{" |  "}}{{color "nc"}}{{searchHighlightMatch $match}}
+				{{- color "search-border"}}{{" |  "}}{{color "nc"}}{{searchHighlightMatch $content $.Query $match}}
```

Comment:
![](https://media1.giphy.com/media/2yA8KXwI7QBykLizf8/giphy.gif?cid=5a38a5a26580f712606e47dc94386ee08485eebc996770da&rid=giphy.gif)


I did not try to optimize around querying the file contents in the case of non-structural search. Just didn't seem like a good use of time relative to effort.